semconv/semconv_test.go

```diff
@@ -24,15 +24,22 @@ import (
 
 func TestAllSemConvFilesAreCrated(t *testing.T) {
 	// Files that have to be present in each semconv package
-	var expectedFiles = []string{"generated_resource.go", "generated_trace.go", "nonstandard.go", "schema.go"}
+	var expectedFiles = []string{"generated_resource.go", "generated_trace.go", "schema.go", "nonstandard.go"}
 
 	files, err := os.ReadDir(".")
 	assert.NoError(t, err)
 	for _, f := range files {
 		if !f.IsDir() {
 			continue
 		}
-		for _, ef := range expectedFiles {
+
+		end := len(expectedFiles)
+		switch f.Name() {
+		case "v1.17.0": // no nonstandard.go file requiered.
```

Comment:
it might not be the ideal solution. But I thought a versions comparison with e.g. `github.com/hashicorp/go-version` would be an overkill. Alternatively we could just add a file without extra constants. 

whats your thoughts on it?